akvo/rsr/management/commands/switch_indicators_to_cumulative_reporting.py

```diff
@@ -0,0 +1,59 @@
+import argparse
+from io import TextIOBase
+from typing import List
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from akvo.rsr.models import Project, Indicator
+from akvo.rsr.models.result.utils import QUANTITATIVE
+
+
+class Command(BaseCommand):
+    help = "Switch all quantitative indicators of the given project and its descendants to cumulative reporting"
+
+    def add_arguments(self, parser: argparse.ArgumentParser):
+        parser.add_argument("project_id", type=int)
+        parser.add_argument(
+            "--exclude", nargs="+", help="ID of indicators that should not be changed"
+        )
```

Comment:
The client has responded and doesn't require this anymore. Due to my other comment (uninherited quantitative indicators), I think the easiest would be to simply remove this option. Otherwise each indicator would have to be checked to ensure that it's not in the descendants of an excluded indicator.

Reply:
Fixed